static/js/controllers/reports.js

```diff
@@ -194,14 +282,17 @@ var _ = require('underscore'),
             $scope.error = false;
             $scope.errorSyntax = false;
             _updateLiveList(data);
```

Comment:
I really don't want LiveList to have to know too much about how Reports works because that's scope creep IMO. Ideally LiveList would deal with rendering the items but not setting state (checking checkboxes, etc). I've broken this rule a little where necessary but I really want to keep this to a minimum. Otherwise all logic/state/etc will end up in ListList and it'll be a great mess.

I've fixed the ignored param.